examples/panel-basic/README.md

```diff
@@ -1,22 +1,26 @@
-# Grafana Panel Plugin Template
+# Grafana Panel Plugin
 
-This is an example on how to build a basic panel plugin.
+This example provides a template for how to build a basic panel plugin. The entry point is `/src/components/SimplePanel/SimplePanel.tsx`.
```

Comment:
I'm not sure this statement is correct. The real entry of the plugin is the `module.ts` but the panel is defined in `SimplePanel.tsx`.

Maybe we can just find a different wording instead of entry point?

Reply:
changed this to "The definition for the panel is provided in `/src/components/SimplePanel/SimplePanel.tsx`."